admin/class-boldgrid-backup-admin-jobs.php

```diff
@@ -337,6 +342,24 @@ public function run() {
 				continue;
 			}
 
+			if ( preg_match('/-(\d{8})-\d{6}\.zip$/', $job['filepath'], $matches ) ) {
```

Comment:
All the logic in this PR, what if we squeeze into into this function? https://github.com/BoldGrid/boldgrid-backup/blob/9780b308f48a09769186a385b3d362bf2395d421/admin/class-boldgrid-backup-admin-jobs.php#L263-L299